### ISSUE TITLE
Create View for Manual Step Recording

### ISSUE THREAD

**Issue by bailee-crandley** [psu-software-engineering-capstone/walktober#17]
As a user I want to have the form for adding in steps accessible, such that I can update my steps manually with ought the use of a fitness tracker.

The view for logging the steps can be done as a overlay that pops up on the profile page to add in the steps. After pressing a sort of + icon to add in steps. Or it can be implemented as another option such as the team chats etc. seen on the side of the app.

This view is necessary to allow for the user to update the number of steps that they have done. As the whole entirety of Walktober centers around the goal of walking a number of steps each day, and for the month.

Thus, we need to implement a form of sorts in a view on the mobile app to allow users to update their total steps. This will include other modules in conjunction with the view.

TLDR: We need a form in the view to allow users to add in steps, with a calculator accessible in it to help them check steps.

Acceptance Criteria

 

- [ ] Screen should maintain consistent styling with other views in the Walktober/PSU brand

- [x] Forum inputs for both the steps taken and the time of the walk

- [ ]  Error checking for invalid numbers entered, when its not in the proper range then inform the user with a redtext/error message.

- [x] Submission button that successfully adds the number of steps entered to the users data/profile
